tests/GoActive.Modules.Geo.Domain.Tests/ValueObjectsTests/TestTitle.cs

```diff
@@ -0,0 +1,26 @@
+﻿using FluentAssertions;
+
+using GoActive.Modules.Geo.Domain.ValueObjects;
+
+namespace GoActive.Modules.Geo.Domain.Tests.ValueObjectsTests;
+
+public class TestTitle
```

Comment:
We still have one not-checked case: the `null` value passed as the argument. 
Let's think how to implement this. 
_Tip: adding new test methods is welcome._